ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -58,5 +65,7 @@ public FrontendConfig(VerifiableProperties verifiableProperties) {
         .getString("frontend.security.service.factory", "com.github.ambry.frontend.AmbrySecurityServiceFactory");
     frontendPathPrefixesToRemove =
         Arrays.asList(verifiableProperties.getString("frontend.path.prefixes.to.remove", "").split(","));
+    frontendChunkedGetResponseThresholdInBytes =
+        verifiableProperties.getInt("ffrontend.chunked.get.response.threshold.in.bytes", 8192);
```

Comment:
typo: you have two "f"s

To avoid such typos, please copy from the config directly. These are quite hard to catch and we are not always looking for them.